service_storagefacility/app/models/storage/event/control/Control.scala

```diff
@@ -0,0 +1,40 @@
+package models.storage.event.control
+
+import models.storage.event.control.ControlAttributes._
+import models.storage.event.{EventType, MusitEvent}
+import no.uio.musit.formatters.WithDateTimeFormatters
+import no.uio.musit.models.{ActorId, EventId, StorageNodeId}
+import org.joda.time.DateTime
+import play.api.libs.json._
+
+/**
+ * TODO: Document me
+ */
```

Comment:
No...this was added way back when...and the code in the "new" packages are copied from the "old" package. Only updating some of the member fields.